src/main/java/software/sigma/bu003/internship/coparts/client/technomir/entity/responce/SupplierInPartFromTehnomir.java

```diff
@@ -0,0 +1,24 @@
+package software.sigma.bu003.internship.coparts.client.technomir.entity.responce;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class SupplierInPartFromTehnomir {
+        private String priceLogo;
+        private Double price;
+        private String currency;
```

Comment:
enum?

Reply:
yes. it`s next ticket, i take it from API